edalize/vcs.py

```diff
@@ -0,0 +1,86 @@
+import os
+import logging
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+class Vcs(Edatool):
+
+    tool_options = {
+        'lists' : {'vcs_options' : 'String'}
+    }
+
+    argtypes = ['plusarg', 'vlogdefine', 'vlogparam']
+
+    MAKEFILE_TEMPLATE = """
+all: $(TARGET)
+
+$(TARGET):
+	vcs -full64 -t $(TOPLEVEL) -f $(TARGET).scr -o $@ $(VCS_OPTIONS)
```

Comment:
I don't know myself around vcs. Is full64 something we would always like to have set, or should that go into VCS_OPTIONS? Or is it something that has to do with 32/64-bit hosts and should be autodetected?

Reply:
```
-full64
   Compiles the design in 64 bit mode and creates a 64 bit executable
   for simulating in 64 bit mode.
```

So it's required on 64 bit platforms (otherwise VCS won't start). Given that I cannot imagine anybody running VCS on a 32 bit machine these days it's probably safe to unconditionally set it.

Detecting 32 vs 64 bit would probably be sufficiently easy, however testing for that is probably annoying ...

Reply:
The rub is that we've always seen vcs fail on 64 bit machines if you don't give the -full64 flag. We could check something but I'm not sure what the best thing to check is?